src/main/resources/assets/megane/lang/zh_cn.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config.waila.megane"                    : "眼镜",
```

Comment:
cool, but i dont think you need to translate the mod name. it just weird, you know :P

Reply:
Alright,I know it's Japanese.

Reply:
yeah, but keep it as "megane" please

Reply:
it's there because hwyla forced to use lang key